drivers/metal/rendering_device_driver_metal.mm

```diff
@@ -2447,6 +2447,10 @@ void deserialize(BufReader &p_reader) {
 	delete obj;
 }
 
+void RenderingDeviceDriverMetal::shader_destroy_modules(ShaderID p_shader) {
+	// TODO.
+}
```

Comment:
FYI @stuartcarnie - for now this is only called by Vulkan code so it being a no-op shouldn't be an issue (I hope :P).